packages/client/components/PokerDiscussVoting.tsx

```diff
@@ -49,16 +49,16 @@ const PokerDiscussVoting = (props: Props) => {
         topLabel = label
       }
     })
-    const scoreLabels = Object.keys(scoreObj).sort((a, b) => {
-      return a === PokerCards.QUESTION_CARD ? -1 : a === PokerCards.PASS_CARD ? 1 : a > b ? -1 : 1
-    })
-    const rows = scoreLabels.map((label) => {
-      return {
-        key: label,
-        scaleValue: scaleValues.find((scaleValue) => scaleValue.label === label)!,
-        scores: scoreObj[label]
-      }
-    })
+    const scoreLabels = Object.keys(scoreObj)
+    const rows = scaleValues
+      .filter((scaleValue) => scoreLabels.find((label) => label === scaleValue.label))
```

Comment:
nice change! i like that you're using their defined sortOrder.
Only 1 change: let's make sure the QUESTION_CARD is at the top & the PASS_CARD is at the bottom. reason being, if someone has a question, that should be the first item of discussion